method/application/create.js

```diff
@@ -0,0 +1,28 @@
+const { isCloakedMsgId: isGroupId } = require('ssb-ref')
+const applicationSpec = require('../../spec/application/application')
+
+module.exports = function CreateGroupApplication (server) {
+  return function createGroupApplication (groupId, authorIds, text, cb) {
+    if (!isGroupId(groupId)) return cb(new Error('expects a valid groupId'))
+    const applicationMessage = {
+      type: 'group/application',
+      version: 'v1',
+      groupId,
+      recps: authorIds,
```

Comment:
we want the recps to be : 
```js
recps: [ ...authorIds, server.id ]
```

This will ensure that the authorIds (the admins of the group) can open it, and also you can open it later

Reply:
So in `accept` just `recps` will be fine, as it'll contain the applicantId as well as the group kaitiakis.

Reply:
Got it, keep conversation on accept.